pkgs/development/libraries/ocl-icd/default.nix

```diff
@@ -23,17 +24,22 @@ stdenv.mkDerivation rec {
     ruby
   ];
 
-  buildInputs = [ opencl-headers ];
+  buildInputs = [ opencl-headers ]
+    ++ lib.optionals stdenv.hostPlatform.isWindows [ windows.dlfcn ]
+  ;
```

Comment:
Nit: `;`on previous line

Reply:
I made that change in order to make future diffs cleaner (so that it's just +1/0 instead of +1/-1). But I'm happy to change it.

Reply:
for that usually  `];` is on same line

Reply:
Okay, fixed

Reply:
you can still make it on new line, `];` usually stick together :-)

Reply:
Yeah but then it's a question of whether you have
```
lib.optionals stdenv.hostPlatform.isWindows [ windows.dlfcn
];
```
or
```
lib.optionals stdenv.hostPlatform.isWindows [
  windows.dlfcn
];
```

personally I'd love it if an autoformatter took over so I didn't have to think about this stuff.

Reply:
yes, it's in the works, we try for formatting not to slowdown merges, it's just some things seem too strange ( read "rare" )